detection_rules/kbwrap.py

```diff
@@ -111,9 +111,10 @@ def kibana_import_rules(ctx: click.Context, rules: RuleCollection, overwrite: Op
 @click.option('--directory', '-d', required=True, type=Path, help='Directory to export rules to')
 @click.option('--rule-id', '-r', multiple=True, help='Optional Rule IDs to restrict export to')
 @click.option('--skip-errors', '-s', is_flag=True, help='Skip errors when exporting rules')
+@click.option('--strip-version', '-sv', is_flag=True, help='Strip the version fields from all rules')
 @click.pass_context
-def kibana_export_rules(ctx: click.Context, directory: Path,
-                        rule_id: Optional[Iterable[str]] = None, skip_errors: bool = False) -> List[TOMLRule]:
+def kibana_export_rules(ctx: click.Context, directory: Path, rule_id: Optional[Iterable[str]] = None,
```

Comment:
please [update](https://github.com/elastic/detection-rules/blob/DAC-feature/CLI.md#uploading-rules-to-kibana) `cli.md` and maybe xref in `docs/custom-rules` that this command should be run with the `CUSTOM_RULES_DIR` envvar set, that way proper validation is applied to versioning when the rules are downloaded